pkg/transform/util/util.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+type SecretDef struct {
+	SecretName string
+	DataKey    string
+}
+
+func SecretRefToSecretDef(secretRef *string) (*SecretDef, error) {
```

Comment:
Well, that is why I created this util under `pkg/transform/`, and it is being used in `object.go` and `kubernetes.go`